package.json

```diff
@@ -27,11 +29,15 @@
     "promise-polyfill": "^7.1.1"
   },
   "devDependencies": {
+    "@types/jest": "^23.3.10",
+    "@types/node": "^10.12.10",
```

Comment:
why do we need these types?

Reply:
Because I import NodeJS.Global and "jest" in index.d.ts. Without those type definitions, tsc would complain when executed in the test script.